misc/docs/source/packages/index.rst

```diff
@@ -46,6 +46,7 @@ The functionality is provided through the following packages:
    obspy.io.nied.knet
    obspy.io.pdas
    obspy.io.reftek
+   obspy.io.resp
```

Comment:
Logically, I think we should move the `NRL` part of this PR over to `clients.nrl`. The `NRL` class feels more like a client than an I/O routine and probably the normal use case will be accessing the NRL library at IRIS via http.